README.md

```diff
@@ -26,19 +26,20 @@ Check out the instructions for [submitting custom metrics from AWS Lambda functi
 
 ## Tracing
 
-Use `ddlambda.AddTraceHeaders(ctx, req)` to inject the Datadog tracing headers to the outbound requests.
+Set the `DD_TRACE_ENABLED` environment variable to `true` to enable Datadog tracing. When Datadog tracing is enabled, the library will inject a span representing the Lambda's execution into the context object. You can then use the included `dd-trace-go` package to create additional spans from the context.
 
-```go
-  req, err := http.NewRequest("GET", "http://example.com/status", nil)
-  // Use the same Context object given to your lambda handler.
-  // If you don't want to pass the context through your call hierarchy, you can use ddlambda.GetContext()
-  ddlambda.AddTraceHeaders(ctx, req)
+```
+import "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 
-  client := http.Client{}
-  client.Do(req)
+func handleRequest(ctx context.Context, ev events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
+  s, _ := tracer.StartSpanFromContext(ctx, "child.span")
```

Comment:
Small thing, but I think usually in go you would write 'defer s.Finish()' right after creating the span, since that guarantees it is called at the end of the function.

Reply:
This is just a code snippet for the README-- the intent was just to illustrate the use of `StartSpanFromContext`. Hypothetically there could be other spans in the same function, and the user would want to have a separate span for each of them. So I think it makes more sense to show the example this way. But I do see your point that if we just wanted the span to include the entire function, then we could use `defer`.